src/main/resources/static/assets/css/icofont.css

```diff
@@ -6022,7 +6022,7 @@
 .icofont-shopping-cart:before {
 	content: "\f0cd";
 }
-.icofont-sign-in:before {
```

Comment:
if you change only the name of this css class please revert these css files, hard to maintain and find issues/bugs in css files, so I suggest the leave them as they were and use the all names.
This is applicable to all css files